pymeasure/instruments/srs/sr830.py

```diff
@@ -421,16 +421,24 @@ def get_scaling(self, channel):
         offset, expand = self.ask("OEXP? %d" % channel).split(',')
         return float(offset), self.EXPANSION_VALUES[int(expand)]
 
-    def set_scaling(self, channel, precent, expand=0):
+    def set_scaling(self, channel, percent=None, expand=None):
```

Comment:
Good catch.

However, please add a parameter `precent=None` for backward compatibility. This parameter should be used, if `percent=None` due to old code. In this case, you should raise a deprecation warning, see #672 for examples.
Please add also a line to deprecation section of the changelog, that `precent` is dperecated in favor of `percent`